src/components/NewBeerForm.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react'
+import axios from 'axios'
+
+export default function NewBeerForm(props) {
+
+    const [value, setValue] = useState(
+        {
+            name: "",
+            tagline: "",
+            description: "",
+            first_brewed: "",
+            brewers_tips: "",
+            attenuation_level: 0,
+            contributed_by: ""
+        }
+    )
+
+    const handleChange = (event) => {
+        const { name, value } = event.target;
+        this.setState({
```

Comment:
estas en un componente funcional, no existe `this.setState`, tiene que ser `setValue`
ademas, para completar el formulario debes copiar el currentState si no cada vez que vos cambies el estado a traves de un input vas a borrar todo lo que hay y solo tendrias el valor de dicho input.

```
setValue({
  ...value,
[name]: value 
})
```

Reply:
Gracias por respuesta @tomasyaya ,Yo estuve probando opciones diferentes y no he notado este fallo :) 
pero eso no ha cambiado nada.. sigo tener el error 500 de servidor. Lo revisare con más tranquilidad despues